### PR TITLE
Added buildscript for request-promise-native for ubuntu_18.04

### PR DESCRIPTION
**Pull request by csrividya**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: r/request-promise-native/request-promise-native-1.0.7_ubuntu_18.04.sh [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+# install dependencies
+apt-get update -y
+apt-get install git curl -y
+
+# install nodejs
+curl https://raw.githubusercontent.com/creationix/nvm/master/install.sh | bash
+export NVM_DIR="$HOME/.nvm"
+[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
+[ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"  # This loads nvm bash_completion
+nvm install 10.24.1
```
Comment (kandarpamalipeddi) [inline r/request-promise-native/request-promise-native-1.0.7_ubuntu_18.04.sh:37]: This installs node version 10.24.1, we are supposed to build and validate on node 12.
can you please check that ?

File: r/request-promise-native/request-promise-native-1.0.7_ubuntu_18.04.sh [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+# ----------------------------------------------------------------------------
+
+set -ex
+
+WORK_DIR=`pwd`
+
+PACKAGE_NAME=request-promise-native
+PACKAGE_VERSION=v1.0.7
```
Comment (kandarpamalipeddi) [inline r/request-promise-native/request-promise-native-1.0.7_ubuntu_18.04.sh:25]: Can you make this PACKAGE_VERSION as configurable ?

**Review by kandarpamalipeddi — CHANGES_REQUESTED** [review]

File: r/request-promise-native/request-promise-native-1.0.7_ubuntu_18.04.sh (hunk 1 above)
Comment (csrividya) [reply r/request-promise-native/request-promise-native-1.0.7_ubuntu_18.04.sh:37]: This package will not work with node 12 . As node-gyp 3.8 is not compatible with node 12, So works for only node version 10

File: r/request-promise-native/request-promise-native-1.0.7_ubuntu_18.04.sh (hunk 2 above)
Comment (csrividya) [reply r/request-promise-native/request-promise-native-1.0.7_ubuntu_18.04.sh:25]: sure I will make this PACKAGE_VERSION as configurable

**Review by kandarpamalipeddi — APPROVED** [review]

**Pull request merged**